@types/index.ts

```diff
@@ -0,0 +1,44 @@
+import { Category } from 'utils/getDataUtils';
+import { Phrase } from 'utils/getDataUtils';
+import { Language } from 'utils/locales';
+
+/** Components Props */
+export type KidsCategoryListProps = {
```

Comment:
Why separate prop types into a separate file?

Reply:
Okay, that seems like a personal preference I have never seen in any codebase I have worked on. 

"For example, no need to import Phase, Language,... in ever component which will use other types."
What is the problem with that? It's not like you need to touch all the places the type is used, you can rename it or change it's definition easily. 

As new, often junior people join the project regularly, it's helpful to stick to default industry conventions, like collocating props with their component, unless there is a strong reason to do otherwise, I don't see one here yet.

Reply:
I have already moved props to components